containers/kubernetes/how-to/upgrade-kubernetes-version-using-cli.mdx

```diff
@@ -0,0 +1,102 @@
+---
+meta:
+title: How to upgrade your Kubernetes Kapsule cluster using the CLI
+description: This page explains how to upgrade your Kubernetes Kapsule cluster using the CLI
+content:
+h1: How to upgrade your Kubernetes Kapsule cluster using the CLI
+paragraph: This page explains how to upgrade your Kubernetes Kapsule cluster using the CLI
+tags: kubernetes kubernetes-kapsule kapsule cluster cli kubectl
+dates:
+categories:
+  - kubernetes
+---
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+    - You have a working [CLI](https://github.com/scaleway/scaleway-cli) with your credentials set up
+    - You are willing to upgrade your Kubernetes Kapsule cluster to the latest k8s version available on the Kapsule API.
+    you're willing to upgrade it.
+</Message>
+
+First, we need to ensure that the latest Kapsule version properly handles your workload. We have a compatibility matrix for different components and your current cluster may use components that are either deprecated or not available on the latest version.
+For more information, you can refer to our [version policy](https://www.scaleway.com/en/docs/containers/kubernetes/reference-content/version-support-policy/).
+
+We recommend you read the Kubernetes [changelog](https://kubernetes.io/releases/) to stay informed on the latest version upgrade.
+
+## Checking which components must be changed
+
+Run the following command into your terminal to retrieve a list of the components that need to be changed.  
+
+`scw k8s version list`
+
+You should get an output similar to the following, providing a list of all relevant components:
+```
+1.25.9   [cilium calico kilo]                [none]                 [containerd]
+1.24.13  [cilium calico weave flannel kilo]  [none]                 [containerd crio]
+1.23.17  [cilium calico weave flannel kilo]  [none nginx traefik2]  [containerd crio docker]
+```
+
+### Ingresses
+
+Managed ingress controllers are deprecated since the minor release 1.24, we provide the feature EasyDeploy instead to set up an ingress controller.
```

Comment:
```suggestion
Managed Ingress Controllers have been deprecated since the minor 1.24 release. As a result, we provide the EasyDeploy feature instead to set up an Ingress Controller. 
```